modules/HapticGlove_module/CMakeLists.txt

```diff
@@ -5,6 +5,24 @@
 # set target name
 set(EXE_TARGET_NAME HapticGloveModule)
 
+if(WALKINGTELEOPERATION_USES_SYSTEM_frugally_deep)
+  find_package(frugally-deep 0.15.19 REQUIRED)
+else()
+  include(FetchContent)
+
+  FetchContent_Declare(
+    functional_plus
+    GIT_REPOSITORY https://github.com/Dobiasd/FunctionalPlus.git
+    GIT_TAG master
+    )
+
+  FetchContent_Declare(
+    frugally_deep
+    GIT_REPOSITORY https://github.com/Dobiasd/frugally-deep.git
+    GIT_TAG master
```

Comment:
⚠️ The latest master of **frugally-deep** does not compile with the following error:
```
home/gromualdi/robot-code/robotology-superbuild/build/src/walking-teleoperation/_deps/frugally_deep-src/include/fdeep/convolution3d.hpp:35:11: error: ‘shape3’ does not name a type; did you mean ‘shape2’?
   35 |     const shape3& filter_shape,
      |           ^~~~~~
      |           shape2
/home/gromualdi/robot-code/robotology-superbuild/build/src/walking-teleoperation/_deps/frugally_deep-src/include/fdeep/convolution3d.hpp:36:11: error: ‘shape3’ does not name a type; did you mean ‘shape2’?
   36 |     const shape3& strides,
      |           ^~~~~~
      |           shape2
/home/gromualdi/robot-code/robotology-superbuild/build/src/walking-teleoperation/_deps/frugally_deep-src/include/fdeep/convolution3d.hpp:37:5: error: ‘padding’ has not been declared
   37 |     padding pad_type,
      |     ^~~~~~~
```


- 🔴 https://github.com/Dobiasd/frugally-deep/commit/b776285b0dd5a6ccf7674ba5ec382ef27b86348a is the first not working commit (associated PR https://github.com/Dobiasd/frugally-deep/pull/375)
- 🟢 is the latest working commit https://github.com/Dobiasd/frugally-deep/commit/cfad37ca2134ab465972311f30fd9d2c583c661e

We should set the commit here
```suggestion
    GIT_TAG cfad37ca2134ab465972311f30fd9d2c583c661e
```